src/kirby/components/empty-state/empty-state.component.ts

```diff
@@ -0,0 +1,12 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'kirby-empty-state',
+  templateUrl: './empty-state.component.html',
+  styleUrls: ['./empty-state.component.scss'],
+})
+export class EmptyStateComponent {
+  @Input() iconName: string;
+  @Input() title: string;
+  @Input() text: string;
```

Comment:
Align naming with `Card` component:
```suggestion
  @Input() subtitle: string;
```

Reply:
Technically it isn't a subtitle. It is more a content, message or text.

Reply:
Fixed